src/test/scala/uk/gov/nationalarchives/consignmentexport/ExternalServiceSpec.scala

```diff
@@ -144,7 +144,7 @@ class ExternalServiceSpec extends AnyFlatSpec with BeforeAndAfterEach with Befor
   def stepFunctionPublish: StubMapping = wiremockSfnServer.stubFor(post(urlEqualTo(stepFunctionPublishPath))
     .willReturn(ok("Ok response body")))
 
-//  val s3Api: S3Mock = S3Mock(port = 8003, dir = "/tmp/s3")
+  //  val s3Api: S3Mock = S3Mock(port = 8003, dir = "/tmp/s3")
```

Comment:
Probably should delete this.

Reply:
This one is a bit odd, I've tried to but doesn't seem to show in the IDE. I think it was added when using the scalaline formatting shortcut.

Reply:
I've deleted this.